assets/src/block-templates/reality-check/template.js

```diff
@@ -1,31 +1,33 @@
 import mainThemeUrl from '../main-theme-url';
 
+const {__} = wp.i18n;
+
 const column = ['core/column', {}, [
   ['core/group', {}, [
     ['core/image', {
       align: 'center',
       className: 'mb-0 force-no-lightbox force-no-caption',
-      url: `${mainThemeUrl}/images/placeholders/placeholder-75x75.jpg`
+      url: `${mainThemeUrl}/images/placeholders/placeholder-75x75.jpg`,
     }],
     ['core/heading', {
       style: {typography: {fontSize: '4rem'}},
       textAlign: 'center',
-      placeholder: __('Enter title', 'planet4-blocks-backend')
+      placeholder: __('Enter title', 'planet4-blocks-backend'),
     }],
     ['core/paragraph', {
       align: 'center',
-      placeholder: __('Enter description', 'planet4-blocks-backend')
+      placeholder: __('Enter description', 'planet4-blocks-backend'),
     }],
     ['core/spacer', {height: '16px'}],
-  ]]
+  ]],
 ]];
 
 const template = () => ([
   ['core/columns', {
-    className: 'block'
+    className: 'block',
   }, [
-    column, column, column
-  ]]
+    column, column, column,
```

Comment:
This is not part of the scope but we can do `[...Array(3).keys()].map(() => column)]` also here!